test/Microsoft.DocAsCode.Build.Engine.Tests/Microsoft.DocAsCode.Build.Engine.Tests.csproj

```diff
@@ -10,6 +10,7 @@
 
   <ItemGroup>
     <PackageReference Include="HtmlAgilityPack" Version="1.4.9" />
+    <PackageReference Include="Moq" Version="4.7.99" />
```

Comment:
ok

---
In reply to: [131612373](https://github.com/dotnet/docfx/pull/1837#discussion_r131612373) [](ancestors = 131612373)